src/containers/Forms/Lesson/CreateLessons.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { Grid, Dialog, DialogContent, Button, DialogTitle, DialogActions } from '@material-ui/core';
+import DateFnsUtils from '@date-io/date-fns';
+import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
+import { ru } from 'date-fns/locale';
+import { registerLocale } from 'react-datepicker';
```

Comment:
Нет пакета в  package.json
```suggestion
import { registerLocale } from 'react-datepicker';
```

Reply:
добавлен

Reply:
Что дает эта утилита, я ее убрал и все так же работает?

Reply:
Мне кажется этот пакет лишний

Убрала